### ISSUE TITLE
A11y_SQLAzureDataStudioMAC_Dashboard_Keyboard: Only color has been used to convey the visual information to the users in the graphs associated with the 'Database Size (MB)' section.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19882]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#Keyboard;#AINotInScope;#WCAG1.4.1;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Right click on the server and select 'Manage'.
5. Dashboard will open.
6. Tab till 'Database Size (MB)' graph section.
7. Use keyboard to access the interactive graph details.

## Actual:
Only color has been used to convey the visual information to the users in the graphs associated with the 'Database Size (MB)' section.

## Expected:
Color should not be used as the only visual means of conveying information to the users. The graphical chart should also infomr the user about what each color refers to and also the color should have structure within it.

## User Impact:
Keyboard users will not know what each color stands for and users with color blindness will not see the color hence structure within the color to be added

## Attachment:

**Comment by bbarik06** [comment]
Since the graph UI is no longer available under the 'Database Size' section. Thus, closing the bug as the new UI that contains a table has no issues.
![MicrosoftTeams-image (10)](https://user-images.githubusercontent.com/98393966/177310751-ba01535b-a775-4530-9ec0-04873e1df79f.png)
